--- pydata-sphinx-theme-0.7.2/pydata_sphinx_theme/__init__.py.orig	2021-11-10 20:51:21.000000000 +0100
+++ pydata-sphinx-theme-0.7.2/pydata_sphinx_theme/__init__.py	2023-01-07 19:29:13.725241701 +0100
@@ -367,7 +367,7 @@ def _get_local_toctree_for(
         kwargs["maxdepth"] = int(kwargs["maxdepth"])
     kwargs["collapse"] = collapse
 
-    for toctreenode in doctree.traverse(addnodes.toctree):
+    for toctreenode in doctree.findall(addnodes.toctree):
         toctree = self.resolve(docname, builder, toctreenode, prune=True, **kwargs)
         if toctree:
             toctrees.append(toctree)
--- pydata-sphinx-theme-0.7.2/pydata_sphinx_theme/bootstrap_html_translator.py.orig	2021-11-10 20:51:21.000000000 +0100
+++ pydata-sphinx-theme-0.7.2/pydata_sphinx_theme/bootstrap_html_translator.py	2023-01-07 19:29:42.538418940 +0100
@@ -1,7 +1,7 @@
 """A custom Sphinx HTML Translator for Bootstrap layout
 """
-from distutils.version import LooseVersion
 from docutils import nodes
+from packaging.version import Version
 
 import sphinx
 from sphinx.writers.html5 import HTML5Translator
@@ -34,10 +34,10 @@ class BootstrapHTML5Translator(HTML5Tran
         # but add 'table' class
 
         # generate_targets_for_table is deprecated in 4.0
-        if LooseVersion(sphinx.__version__) < LooseVersion("4.0"):
+        if Version(sphinx.__version__) < Version("4.0"):
             self.generate_targets_for_table(node)
 
-        if LooseVersion(sphinx.__version__) < LooseVersion("4.3"):
+        if Version(sphinx.__version__) < Version("4.3"):
             self._table_row_index = 0
         else:
             self._table_row_indices.append(0)
--- pydata-sphinx-theme-0.7.2/setup.py.orig	2021-11-10 20:51:21.000000000 +0100
+++ pydata-sphinx-theme-0.7.2/setup.py	2023-01-07 19:32:33.520825982 +0100
@@ -48,7 +48,7 @@ setup(
     include_package_data=True,
     # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
     entry_points={"sphinx.html_themes": ["pydata_sphinx_theme = pydata_sphinx_theme"]},
-    install_requires=["sphinx", "beautifulsoup4", "docutils!=0.17.0"],
+    install_requires=["sphinx", "beautifulsoup4", "docutils!=0.17.0", "packaging"],
     extras_require={
         "test": tests_require,
         "coverage": ["pytest-cov", "codecov", *tests_require],
